### DISCUSSION TITLE
Weekly log

### DISCUSSION THREAD

**Discussion by silentHoo** [inovex/elements#466] [General]
Collection of logs of our discussion, every week.

**Comment by silentHoo** [comment]
# 2021-11-29

* Issues from discussion in #464 created?

**Comment by janivo** [comment]
# 2021-12-06
- #377 - Icon `request` should be renamed to `share`
- #457 - Continue working on the editor
- #472 - will be addressed by @BenPag

**Comment by JCofman** [comment]
# 2021-12-13
- [autocomplete component](https://elements.inovex.de/dist/latest/storybook/?path=/docs/input-ino-autocomplete--playground) has some (performance bottlenecks, cannot be used with specific patterns)
- Composable pattern example (e.g., autocomplete)
- Global Styles (examples... just some usage of CSS classes with basic HTML nodes)

# ToDos
- create a basic example with angular and composition of `ino-input` `ino-options` to create an autocomplete 


# Some research input
- Downshift (autocomplete example) https://github.com/downshift-js/downshift
- https://turborepo.org/ for caching monorepo
- Storybook search would be awesome maybe -> [meili search](https://www.meilisearch.com/)

**Comment by janivo** [comment]
# 2022-01-17
- Since v6.4. Storybook supports Code-Snippets with WebComponents
- Therefore we can switch over to use the [Storybook Args](https://storybook.js.org/docs/react/writing-stories/introduction#using-args) which reduces our boilerplate code significantly
- https://github.com/inovex/elements/pull/490 shows a first example
- Next steps are:
  -  finish https://github.com/inovex/elements/pull/490
  - Migrate stories to new arg format
  - Move prop related Readme content to the jsdoc in the component
  - The Readme should be kept minimal

**Comment by pfecht** [comment]
# 2022-01-31
- No developer available, thus not discussed any dev topics
- However, discussed with Designers (and Jan G. and Maik) 
  - Designers start a new Adobe XD project (based on the old one) and want to rework all compoents to an actual library.
  - Jan will further talk to us to merge the "composite library" of inma with our angular wrapper.
